.env-sample

```diff
@@ -0,0 +1,14 @@
+# Set to 'production' when deployed
+RACK_ENV=development
+
+MONGO_URI=
```

Comment:
`PAGESNAP_URL` is one that’s missing here. That said, neither it nor `MONGO_URI` are required… they both will be filled in with default values.